core/src/client/mod.rs

```diff
@@ -1005,7 +1004,7 @@ impl KakarotClient for KakarotClientImpl {
             }
         }
 
-        let kakarot_class_hash = FieldElement::from_hex_be(KAKAROT_CONTRACT_ACCOUNT_CLASS_HASH)
+        let kakarot_class_hash = FieldElement::from_hex_be(ACCOUNT_CLASS_HASH)
```

Comment:
How do you see changing the name of the variable **kakarot_class_hash** to **account_class_hash**?

Reply:
I think @ClementWalter already made this change in #125

Reply:
He actually [remove the constants](https://github.com/sayajin-labs/kakarot-rpc/pull/125/files#diff-3f796ff7496f37e2f6e670ee3eea1fe7a4fb28955ab6bb1867b60037f593a0f6L4), maybe this PR should be closed :thinking:

Reply:
Good point, you're right